general/administration/installing.md

```diff
@@ -218,6 +218,64 @@ Navigate to Playback tab in the Dashboard and set the path to FFmpeg under FFmpe
 
 Generic amd64 Linux builds in TAR archive format are available [here](https://jellyfin.org/downloads/#linux).
 
+### Installation Process
+
+Create a directory in /opt for jellyfin and it's files, and cd into it. The following assumes the directory is called "jellyfin"
+    
+    sudo mkdir /opt/jellyfin 
+    cd /opt/jellyfin
+
+Download the latest generic linux build from the [Jellyfin release page](https://github.com/jellyfin/jellyfin/releases). The generic linux build ends with "linux-amd64.tar.gz". The rest of these instructions assume version 10.4.3 is being installed (i.e. jellyfin_10.4.3_linux-amd64.tar.gz). Download the generic build, then extract the archive:
+
+    sudo wget https://github.com/jellyfin/jellyfin/releases/download/v10.4.3/jellyfin_10.4.3_linux-amd64.tar.gz
+    sudo tar xvzf jellyfin_10.4.3_linux-amd64.tar.gz
+
+Create a symbolic link to the Jellyfin 10.4.3 directory. This allows an upgrade by repeating the above steps and enabling it by simply re-creating the symbolic link to the new version.
+
+    sudo ln -s jellyfin_10.4.3 jellyfin
+
+Create four sub-directories for Jellyfin data:
+    
+    sudo mkdir data cache config log
+
+If you are running Debian or a derivative, you must also download and install the Jellyfin specific ffmpeg. Be sure to download the latest release that matches your OS (4.2.1-3 for Debian Stretch assumed below).
+
+    sudo wget https://github.com/jellyfin/jellyfin-ffmpeg/releases/download/v4.2.1-3/jellyfin-ffmpeg_4.2.1-3-stretch_amd64.deb
+    sudo dpkg --install jellyfin-ffmpeg_4.2.1-3-stretch_amd64.deb
+    
+If you run into any dependency errors, run this and it will install them and jellyfin-ffmpeg:
+
+    sudo apt install -f
+
+    
+Due to the number of command line options that must be passed, it is easiest to create a small script (here called jellyfin.sh) to run Jellyfin.
+
+Type:
+    
+    sudo nano jellyfin.sh
+    
+Then paste the following and modify and needed:
+
+    #!/bin/bash
+    JELLYFINDIR="/opt/jellyfin"
+    FFMPEGDIR="/usr/share/jellyfin-ffmpeg"
+
+    $JELLYFINDIR/jellyfin/jellyfin \
+    -d $JELLYFINDIR/data \
+    -C $JELLYFINDIR/cache \
+    -c $JELLYFINDIR/config \
+    -l $JELLYFINDIR/log \
+    --ffmpeg $FFMPEGDIR/ffmpeg 
```

Comment:
The 4 spaces is the markdown for a code block

Reply:
I'd prefer the explicit style blocks, since it tells the system (and the editor) what language it is.
```md
```bash
#!/bin/bash
```

And if the content comes from a file, please use these: https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a#code-snippet